public/static/docs/command-reference/remote/modify.md

```diff
@@ -296,6 +296,28 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
   ```dvc
   $ dvc remote modify myremote credentialpath /path/to/my/creds/[FILE_NAME].json
   ```
+  
+  Google Cloud is migrating to allowing only Service Accounts for application logins. To ensure forward compatibility and better security - its best practice to set up a Goole Sevice account for your project. You can follow these steps:
+  
+  - [Create a GCloud Service account for the `projectname` above](https://www.google.com/search?client=safari&rls=en&q=gcloud+service+account&ie=UTF-8&oe=UTF-8).
+
+  - Ensure the account has read and write access to any bucket resource hosting the remote `url` you set up above. Best practice is to limit the access credentials to only what the DVC service requires. You can check out the IAM roles for `Storage Object Creator` and `Storage Object Viewer` which limit an account to only create and view .. you guessed it - `Storage Objects`.
+  
+  - Download the JSON credentials for your newly created service account and store them somewhere safe.
+  
+  - Ensure you habve the [gcloud command line tools installed.](https://cloud.google.com/sdk/docs/quickstarts)
```

Comment:
minor: habve -> have

Reply:
I did this PR via Github web UI, is there an easy way to change my code without doing a checkout and committing to the branch via a real machine? Just curious. Good catch!

Reply:
I think you can just click edit button:

<img width="1135" alt="Screen Shot 2020-02-29 at 1 53 03 PM" src="https://user-images.githubusercontent.com/3659196/75615609-d4fe9780-5afa-11ea-858c-ce29168989ed.png">

Reply:
LOL 
<- RTFM newb

Thanks. Ill make this more clear.